ipa/verifier.go

```diff
@@ -24,6 +24,7 @@ func CheckIPAProof(transcript *common.Transcript, ic *IPAConfig, commitment band
 
 	w := transcript.ChallengeScalar("w")
 
+	// Rescaling of q.
```

Comment:
I think can help future readers on how this maps to a spec explanation of this security step.